rpcs3/Emu/RSX/Overlays/overlay_controls.h

```diff
@@ -880,6 +881,18 @@ namespace rsx
 				return compiled_resources;
 			}
 
+			//! Width including padding
+			u32 get_total_width() const
+			{
+				return static_cast<u32>(w + padding_left + padding_right);
```

Comment:
padding should be inside the width, margin outside the width. i.e w includes padding